ScriptEngine/src/api/EntityAPI.cpp

```diff
@@ -983,6 +992,168 @@ Local<Value> EntityClass::setHealth(const Arguments& args) {
     CATCH("Fail in setHealth!");
 }
 
+Local<Value> EntityClass::setAbsorption(const Arguments& args) {
+    CHECK_ARGS_COUNT(args, 1);
+    CHECK_ARG_TYPE(args[0], ValueKind::kNumber);
+
+    try {
+        Actor* entity = get();
+        if (!entity)
+            return Local<Value>();
+
+        AttributeInstance* AbsorptionAttribute = entity->getMutableAttribute(Global<SharedAttributes>->ABSORPTION);
```

Comment:
Have you viewed our contributing note?
This is non-standard naming, please use lower camel case
same below